apps/condo/domains/ticket/utils/triggers.js

```diff
@@ -0,0 +1,71 @@
+const get = require('lodash/get')
+
+const { getByCondition } = require('@core/keystone/schema')
+
+const { COUNTRIES, DEFAULT_LOCALE } = require('@condo/domains/common/constants/countries')
+
+const { TICKET_ASSIGNEE_CONNECTED_TYPE, TICKET_EXECUTOR_CONNECTED_TYPE } = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+
+const triggerTicketEventActions = async ({ operation, existingItem, updatedItem, context }) => {
```

Comment:
It's better to move all such things to the worker. May be in a future (optional)